commit d058d4e36490a9725a2da20c5ba23a28e559e4a9
Author: Daniel Kolesa <daniel@octaforge.org>
Date:   Fri Sep 2 19:33:38 2022 +0200

    increase the file number limit for adb
    
    At least in kernel packages for aarch64 we were running into
    the 8000 file hard limit in linux-devel. Increase it until we
    have a better solution in place.

diff --git a/src/apk_adb.h b/src/apk_adb.h
index 5b8283c..e3ee5de 100644
--- a/src/apk_adb.h
+++ b/src/apk_adb.h
@@ -93,7 +93,7 @@ struct adb_data_package {
 #define APK_MAX_PKG_REPLACES		32
 #define APK_MAX_PKG_TRIGGERS		32
 #define APK_MAX_INDEX_PACKAGES		20000
-#define APK_MAX_MANIFEST_FILES		8000
+#define APK_MAX_MANIFEST_FILES		12000
 #define APK_MAX_MANIFEST_PATHS		6000
 
 extern const struct adb_object_schema
